.github/workflows/ci.yml

```diff
@@ -40,10 +40,6 @@ jobs:
           - {moodle-branch: 'MOODLE_39_STABLE', php: '7.3', node: '14.15', database: 'pgsql'}
           - {moodle-branch: 'MOODLE_39_STABLE', php: '7.4', node: '14.15', database: 'mariadb'}
           - {moodle-branch: 'MOODLE_39_STABLE', php: '7.4', node: '14.15', database: 'pgsql'}
-          - {moodle-branch: 'MOODLE_310_STABLE', php: '7.4', node: '14.15.1', database: 'mariadb'}
```

Comment:
Instead of removing these, just convert this whole ci over to the latest version and touch version.php ->supported

Reply:
Hi @brendanheywood 

I have replaced the ci. Would you please have another look one you have time? Thanks